### PR TITLE
Implement netpbm format

### PR DESCRIPTION
**Pull request by martinkauppinen**

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -1,27 +1,155 @@
...
+    fprintf(stderr,
+            "Usage:\n"
+            "\t%s [options] <filepath> [red] [green] [blue]\n"
+            "Options:\n"
+            "\t-x, --xbm    Print an xbm bitmap (default)\n"
+            "\t-n, --netpbm Print a netpbm bitmap\n"
+            "\t-h, --help   Show this help\n"
+            "Colors:\n"
+            "\tSupply integer arguments in the range 0-255 for\n"
+            "\tthe red, green, and blue channels. Only applies to\n"
```
Comment (kjeller) [inline src/main.c:35]: `Only applies to bitmaps that do not support color` should be `Only applies to bitmaps that support color`

**Review by kjeller — COMMENTED** [review]
Looks swell my dude, thanks for cleaning everything up nicely. 🥇
I am not that familiar with the netbmp format but I tried compiling and running this PR and it works flawlessly.

Though, there is an inconsistency between some files ending with a newline and some that don't. I think this is ready to merge after my comments are resolved and this inconsistency is fixed.

File: src/main.c (hunk 1 above)
Comment (martinkauppinen) [reply src/main.c:35]: Nope, only to bitmaps that do not support color. The supplied RGB triple is used as the color. Bitmaps with color support ignore the RGB arguments, as they have their own color embedded in the file.

Comment (kjeller) [reply src/main.c:35]: Ohyea.. right. I interpreted that wrong 😂

**Pull request merged**